### DISCUSSION TITLE
Creating interactive GitHub profile

### DISCUSSION THREAD

**Discussion by ghost** [community/community#79150] [New to GitHub]
Here I am trying to create a repository with the name of my profile. But Here It gives an error. But there is no already created repository with that name. Do you know how to fix this error?

![Screenshot (1698)](https://github.com/community/community/assets/116453860/416fd539-3abb-44be-a4e6-702f30dccfb6)

**Comment by MantasTech** [comment]
Hi Chanaka,

Try to check if you don't have this repo already as private.
If it's not the case then your best option is to contact support by creating a new [ticket.](https://support.github.com/request)
Might be some naming restrictions or reserved name issue.

Good luck, and I hope this gets resolved quickly for you!

**Reply by ghost** [reply]
Thank you for answering MantasTech. 

No repo is private. So I will contact the support team.
